styles/01-css-approaches.mdx

```diff
@@ -0,0 +1,111 @@
+## CSS approaches
+
+* CSS is all in the global scope by default
+* CSS-in-JS approaches with React let you scope your CSS to individual components
+* Functional CSS approaches let you avoid writing redundant CSS
+
+---
+
+## CSS-in-JS
+
+* Approaches that move some part of the CSS process into JavaScript
+* Normally try to solve the problem of CSS being in the global scope
+* Can require more set up, but end up more flexible in the long term
+* Read into CSS Modules and Styled Components if this method is of interest
```

Comment:
Do you have a list of resources you've found useful?

Can include them here - or as part of the readme of this repo - a collection of additional reading / resources / etc.

Reply:
I'll add to the readme.